externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/SparkTBinaryFrontendService.scala

```diff
@@ -92,6 +92,10 @@ class SparkTBinaryFrontendService(
       None
     }
   }
+
+  override def extraServiceInfo: Map[String, String] = {
```

Comment:
why is spark special?

Reply:
<img width="859" alt="image" src="https://user-images.githubusercontent.com/6757692/197497450-0df9f7e4-21ce-4268-978d-17fdabb4dd1f.png">

Now I can only see the `KYUUBI_ENGINE_ID` is used in Spark.

We can extend other engines to support expose kyuubi.engine.id.

Reply:
why extra is added? do we have an info method?

Reply:
> why extra is added? do we have an info method?

I did not get it.

Reply:
The extra is to distinguish w/ the service host/port information, if it's confusing, I suggest naming it attributes here.